esp/scm/ws_resources.ecm

```diff
@@ -25,11 +25,35 @@ ESPStruct HPCCService
     bool TLSSecure;
 };
 
+ESPStruct ServicePorts
+{
+    string Name;
+    string Protocol;
+    unsigned Port;
+    unsigned NodePort;
+    unsigned TargetPort;
+};
```

Comment:
What are the 3 ports?  I don't think there is any need to know any of the "internal" ports info?

Reply:
I think you can remove the internal TargetPort at least until someone asks for it.  Whether anyone will use the NodePort if the type is loadbalancer I'm not sure.

Reply:
I could be wrong, but I think the NodePort should be used if the type is "NodePort" (not "LoadBalancer"). See:https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/pkg/apis/core/types.go#L3475

Reply:
@wangkx from the clients point of view, there isn't any need to report on any IPs or Ports that are "inside the cluster" - however ops may want it in the topology info?